### DISCUSSION TITLE
Calibre 5.44 and NoDRM 10.0.2

### DISCUSSION THREAD

**Discussion by mrcmrc12** [noDRM/DeDRM_tools#93] [General]
I'm having trouble with Calibre 5.44 and NoDRM 10.0.2 installed on a new Mac Studio (Monterey 12.4): just the .azw3 ebooks imported still have the DRM... (the Kindle app for  MacOS is installed and I can read the ebook of my library).

I also do not undrestand how to run Calibre in debug mode to put here the log... Under "Preferences" I don't see an option to do it.

Any help? Thanks!
- Marco.

**Comment by ElleKayEm** [comment]
Was it downloaded from the website for an E-ink Kindle?  Make sure you have the Kindle's serial number entered correctly into DeDRM's customization.

To get a log, click the drop-down arrow next to the preferences button and then "restart in de-bug mode".  Add the book, close calibre, and the log will open.

**Reply by mrcmrc12** [reply]
Thanks! Was the missing serial number! I thought it was enough to insert the key file...

- Marco.

**Reply by ElleKayEm** [reply]
Each Kindle app installation and device has a unique key even if they are logged in to the same account.
